client/components/Cart.js

```diff
@@ -0,0 +1,47 @@
+// LIBRARIES
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { getArtworks } from '../redux/artworks';
+
+export class Cart extends Component {
+  render() {
+    const {artworks, cart} = this.props
+    //if you go directly to the cart page, you don't have the artwork, so this is a workaround but we probably need to figure out the right way to do this later
+    if(artworks.length === 0) {
+      this.props.getArtworks();
+      return (<div>Loading</div>)
```

Comment:
This isn't the cart artwork, this is *all* the artwork. I'm just getting it here so that I can look up stuff like title, artist, pic, etc., in order to render a nice cart. The cart only stores the artworkId and quantity (which was probably a mistake).